boards/arm/stm32f429i_disc1/doc/stm32f429i_disc1.rst

```diff
@@ -0,0 +1,203 @@
+.. _stm32f429i_disc1_board:
+
+ST STM32F429I Discovery
+#######################
+
+Overview
+********
+
+The STM32F429I Discovery kit features an ARM Cortex-M4 based STM32F429ZI MCU
+with a wide range of connectivity support and configurations Here are
+some highlights of the STM32F429I-DISC1 board:
+
+
+- STM32 microcontroller in LQFP144 package
+- Extension header for all LQFP144 I/Os for quick connection to prototyping board and easy probing
+- On-board ST-LINK/V2-B debugger/programmer with SWD connector
+- Flexible board power supply:
+
+       - ST-LINK/V2-1 USB connector
+       - User USB FS connector
+       - VIN from Arduino* compatible connectors
+
+- Two push-buttons: USER and RESET
+- USB OTG FS with micro-AB connector
+- 2.4-inch QVGA LCD with MIPI DSI interface and capacitive touch screen
+- 64Mbit SDRAM
+- L3GD20, ST-MEMS motion sensor 3-axis digital output gyroscope
+- Six LEDs
+
+	- LD1 (red/green) for USB communication
+	- LD2 (red) for 3.3 V power-on
+	- Two user LEDs: LD3 (green), LD4 (red)
+	- Two USB OTG LEDs: LD5 (green) VBUS and LD6 (red) OC (over-current)
+
+.. image:: img/en.stm32f429i_disc1.png
+     :width: 350px
+     :align: center
+     :height: 631px
+     :alt: STM32F429I-DISC1
+
+More information about the board can be found at the `STM32F429IDISCOVERY website`_.
+
+Hardware
+********
+
+STM32F429I-DISC1 Discovery kit provides the following hardware components:
+
+- STM32F429ZIT6 in LQFP144 package
+- ARM®32-bit Cortex®-M4 CPU with FPU
+- 180 MHz max CPU frequency
+- VDD from 1.8 V to 3.6 V
+- 2 MB Flash
+- 256+4 KB SRAM including 64-Kbyte of core coupled memory
+- GPIO with external interrupt capability
+- 3x12-bit ADC with 24 channels
+- 2x12-bit D/A converters
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (17)
+- Watchdog Timers (2)
+- USART/UART (4/4)
+- I2C (3)
+- SPI (6)
+- SDIO
+- 2xCAN
+- USB 2.0 OTG FS with on-chip PHY
+- USB 2.0 OTG HS/FS with dedicated DMA, on-chip full-speed PHY and ULPI
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F429ZI can be found here:
+       - `STM32F429ZI on www.st.com`_
+       - `STM32F429 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f429i_disc1 board configuration supports the following hardware features:
```

Comment:
You should be consistent in how you write stm32f429i_disc1 board throughout the doc (some references are all upper case, some without the underscore).